Bio/EBI/ebisearch.py

```diff
@@ -0,0 +1,691 @@
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+"""Provide code to access the EBI Search API.
+
+This module aims to make EBI Search API easy to use. See:
+https://www.ebi.ac.uk/ebisearch/overview.ebi
+
+The EBI Search API is a text search engine to access biological data resources
+hosted at the European Bioinformatics Institute (EMBL-EBI).
+
+The following code was developed by Berenice Batut (berenice.batut@gmail.com)
+
+References:
+Park Y.M., Squizzato S., Buso N., Gur T., Lopez R. (2017)
+The EBI search engine: EBI search as a service making biological data accessible
+for all.;Nucleic Acids Research, May 2, 2017; DOI: 10.1093/nar/gkx359
+
+"""
+
+import requests
+
+import requires_internet
+requires_internet.check()
+
+baseUrl = 'http://www.ebi.ac.uk/ebisearch/ws/rest'
+sizeLimit = 100
+startLimit = 250000
+
+
+def get_domain_details(domain):
+    """Return a dictionary with the details of a given domain in EBI.
+
+    >>> get_domain_details("allebi")
+
+    domain: domain id in EBI
```

Comment:
It is not explicit that this is one of the arguments (but we have not settled on a common preferred reStructuredText style for this). I would suggest using the bullet point list style in ``Bio/Seq.py`` or the very explicit ``:param domain: domain ID in EBI`` style used in ``Bio/Phylo``.